post_install.sh

```diff
@@ -19,7 +19,7 @@ pw useradd \
 
 mkdir -p "$INSTALL_DIR"
 git clone "$GIT_URL" "$INSTALL_DIR"
-(cd "$INSTALL_DIR" && npm ci --production && npm run build)
+(cd "$INSTALL_DIR" && npm ci && npm run build)
```

Comment:
Another option is to base this off the [zigbee2mqtt Dockerfile](https://github.com/Koenkk/zigbee2mqtt/blob/master/docker/Dockerfile#L15) though I'm not sure if there is any advantage to this

```
cd $INSTALL_DIR \
  && npm ci --no-audit --no-optional --no-update-notifier \
  && npm run build \
  && rm -rf node_modules
  && npm ci --production --no-audit --no-optional --no-update-notifier && \
```

Reply:
For the Dockerfile it's probably helpful for reducing the layer size, but I don't think it's too important for this use case. Older versions of the z2m docs included `--production`, but since the switch over to Typescript just `npm ci` is the recommended method.